src/Linux/Avalonia.LinuxFramebuffer/LinuxDrmOptions.cs

```diff
@@ -0,0 +1,46 @@
+using Avalonia.LinuxFramebuffer.Output;
+using Avalonia.Media;
+using JetBrains.Annotations;
+
+namespace Avalonia.LinuxFramebuffer
+{
+    public class LinuxDrmOptions
```

Comment:
IMO this class currently violates single responsibility principle: it's passed to DrmOutput's constructor as an argument and can have a predefined `IOutputBackend OutputBackend` property at the same time.

It also controls both the target device (`Card`), connector discovery (`DrmConnectorsForceProbe`) and output-specific options `Scaling`,  `EnableInitialBufferSwapping`, `InitialBufferSwappingColor` that can be used separately when DrmOutput is initialized with manually specified connector and mode info.



IMO that should be split into 
`DrmOutputOptions`
- `Scaling`
- `EnableInitialBufferSwapping`
- `InitialBufferSwappingColor`

and extra parameters for `DrmOutput` constructor. So the DrmOutput constructors look be:

- `public DrmOutput(DrmCard card, DrmResources resources, DrmConnector connector, DrmModeInfo modeInfo, DrmOutputOptions? options = null)`
- `public DrmOutput(DrmOutputOptions? options = null, string? card = null, bool forceProbeConnectors = false)`